podcasts/PodcastViewController.swift

```diff
@@ -149,6 +152,10 @@ class PodcastViewController: FakeNavViewController, PodcastActionsDelegate, Sync
     @IBOutlet var multiSelectHeaderView: ThemeableView!
     private let operationQueue = OperationQueue()
 
+    // Constraint to adjust multiSelectHeader based on device size
+    @IBOutlet weak var multiSelectHeaderViewConstraint: NSLayoutConstraint!
+    private let screenWidth = UIScreen.main.bounds.width
```

Comment:
This feels out of place here.

Instead of having this here, a better idea would be to extract the `self.screenWidth < 380.0 ? 65.0 : 90.0` logic to a method, keeping `UIScreen.main.bounds.width` inside it.

Reply:
Extracted logic to a method to better cover current and future edge cases.

Tested on smaller devices:
- iPod Touch (320)
- iPhone SE (375)

![320 - 375 (iPod - Iphone SE)](https://user-images.githubusercontent.com/81433983/227676370-0c7d9f46-9dfb-4aff-a347-ff464d8854dd.png)

Tested on iPads:
- iPad Mini 4 (768)
- iPad Pro 6th Gen (1024)

![iPad 768 - 1024](https://user-images.githubusercontent.com/81433983/227676911-3e31b5f1-5429-4620-962e-2fbb8ba44abc.png)